change/@ni-jasmine-parameterized-5d135b15-d9c6-410c-8cba-5784b0eda870.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@jattasNI wanna nip this typo in the README breaking syntax highlighting?

![image](https://github.com/ni/nimble/assets/1588923/6d83ee66-aae5-4deb-9a1b-856c419594c0)